clang/lib/Basic/SourceManager.cpp

```diff
@@ -458,11 +458,14 @@ SourceManager::AllocateLoadedSLocEntries(unsigned NumSLocEntries,
       CurrentLoadedOffset - TotalSize < NextLocalOffset) {
     return std::make_pair(0, 0);
   }
-  LoadedSLocEntryTable.resize(LoadedSLocEntryTable.size() + NumSLocEntries);
-  SLocEntryLoaded.resize(LoadedSLocEntryTable.size());
+
+  unsigned NewTableSize = LoadedSLocEntryTable.size() + NumSLocEntries;
```

Comment:
You're storing `size_t` here, but the value is calculated as `unsigned`.  Not sure which is correct

Reply:
I think `size_t` fits better, I'll use that in the next revision.